Classes/Neos/Neos/Ui/FlowQueryOperations/NeosUiDefaultNodesOperation.php

```diff
@@ -55,28 +55,44 @@ public function canEvaluate($context)
      */
     public function evaluate(FlowQuery $flowQuery, array $arguments)
     {
-        list($site, $documentNode) = $flowQuery->getContext();
-        list($baseNodeType, $loadingDepth) = array_replace([
+        list($siteNode, $documentNode) = $flowQuery->getContext();
+        // set default values for arguments
+        list($baseNodeType, $loadingDepth, $toggledNodes) = array_replace([
```

Comment:
That looks risky (or I don't get it). If $arguments contains 3 values the array replace could be skipped. If it contains less, How do you know here which arguments are set?

Reply:
Agreed, it was not my code, took me some time to figure out what it's doing actually. I think defaults are not needed here, will remove it.